docs/src/markdown-pages/aptitudes/user.createJWT.md

```diff
@@ -2,5 +2,20 @@
 name: "JWT"
 links_js: "jwt"
 ---
-Receive a [JWT](https://jwt.io) with the currently logged in Olive Helps user's email address in the `subject` claim, the Loop ID in the `audience` claim, and a 5-minute `expiration`.
+Receive a [JWT](https://jwt.io) identifying the currently logged in Olive Helps user.
 
+## JWT Claims
+The claims in the JWT are as follows:
+
+Standard Claims:
+`sub` (Subject): contains the unique User ID of the current Olive Helps user.
```

Comment:
Markdown treats single new lines as standard space so they'll run together. To present them in a list like this you'll probably want to format them as markdown lists, or maybe as a table.

Reply:
Fixed.